src/pages/Login/Login.css

```diff
@@ -6,6 +6,7 @@
         background-repeat: no-repeat;
         background-size: cover;
         height: 100%;
+        min-height: 90vh;
```

Comment:
`If you remove min-height you will get the ugly background color of white at the bottom of the page. Where do you white at the bottom of the page.` I understand that but by modifying Login.css you also affected 5 forms. 

This global rule affects many forms here, Try using that rule on the file you are working on. Instead of applying globally. To 5 forms.
1) AddFriendForm.js - position is changed on a desktop screen.
2) ForgotPassword.js - No changes
3) Login.js, - No changes
4) ResetPassword - is affected by changes
5) Signup.js - This is affected by changes.

Reply:
I've fixed the issue. I'm sorry for not realising that. 
I've added a local style.